v19.2/vectorized-execution.md

```diff
@@ -0,0 +1,58 @@
+toc: true
+---
+
+ <span class="version-tag">New in v19.2:</span> CockroachDB supports column-oriented ("vectorized") query execution on all [supported SQL operations](https://www.cockroachlabs.com/docs/stable/sql-feature-support.html).
+
+ Many SQL databases, including MySQL, PostgreSQL, and all versions of CockroachDB released prior to 19.2, execute [query plans](https://en.wikipedia.org/wiki/Query_plan) one row of table data at a time. The CockroachDB vectorized execution engine dramatically improves performance over row-oriented execution by processing each component of a query plan on type-specific batches of column data. Column-oriented execution models contrast with row-oriented execution models, which can offer good performance for [online transaction processing (OLTP)](https://en.wikipedia.org/wiki/Online_transaction_processing) queries, but suboptimal performance for [online analytical processing (OLAP)](https://en.wikipedia.org/wiki/Online_analytical_processing) queries.
+
+
+## Configuring vectorized execution
+
+By default, vectorized execution is enabled in CockroachDB for all supported "streaming" SQL operations (i.e. operations that do not require [memory buffering](https://en.wikipedia.org/wiki/Data_buffer)). Non-streaming operations include global [sorts](query-order.html), [window functions](window-functions.html), [hash joins](joins.html#hash-joins), and [merge joins](joins.html#merge-joins) on non-unique columns. Merge joins on columns that are guaranteed to have one row per value, also known as "key columns", are considered streaming operations.
+
+You can turn vectorized execution on or off for all operations with the `sql.defaults.vectorize` [cluster setting](cluster-settings.html) or the `vectorize` [session variable](set-vars.html). The following options are supported for both the cluster setting and session variable:
+
+Option | Description
+----------|------------
+`auto` | Instructs CockroachDB to use the column-oriented vectorized SQL execution engine on streaming operations. By default, `vectorize=auto`.
+`experimental_on` | Turns vectorization on for all supported operations. We do not recommend using this setting in production environments, as it can lead to memory issues. See [Known Limitations](vectorized-execution.html#known-limitations) for more information.
+`off` | Turns vectorization off for all operations.
+
+You can verify whether will use the execution engine or not with the [`EXPLAIN`](explain.html) statement.
+
+For more information about configuring cluster settings, see [Cluster Settings](cluster-settings.html). For more information about setting session variables, see [`SET` &lt;session variable&gt;](set-vars.html).
+
+### Setting the row threshold for vectorized execution
+
+The efficiency of column-oriented execution increases with the number of rows processed. If you are querying a table with a small number of rows, it is currently more efficient to use row-oriented execution.
```

Comment:
I think we should standardize on one usage everywhere in our documentation for this.  Otherwise it's going to be hard to find stuff about vectorized.  We could either say "vectorized" or say "column-oriented" or say "column-oriented ('vectorized')" but I don't think we should mix and match all 3.  The first mention of vectorized on each page where it appears could be "vectorized (column-oriented) execution" and thereafter just use "vectorized".